Cargo.lock

```diff
@@ -1105,9 +1061,12 @@ dependencies = [
 
 [[package]]
 name = "error-chain"
```

Comment:
hrm, I thought we got rid of `error-chain` completely. Seems to related to `ethabi` though.